commands/command.go

```diff
@@ -36,12 +36,22 @@ var CommandListCmd = &cobra.Command{
 }
 
 var CommandDeleteCmd = &cobra.Command{
-	Use:     "delete",
-	Short:   "Delete a slash command",
-	Long:    `Delete a slash command. Commands can be specified by command ID.`,
-	Example: `  command delete commandID`,
+	Use:        "delete",
+	Short:      "Delete a slash command",
+	Long:       `Delete a slash command. Commands can be specified by command ID.`,
+	Example:    `  command delete commandID`,
+	Deprecated: "please use \"archive\" instead",
+	Args:       cobra.ExactArgs(1),
+	RunE:       withClient(archiveCommandCmdF),
+}
+
+var CommandArchiveCmd = &cobra.Command{
+	Use:     "archive",
```

Comment:
In most of the other CLI docs, when a option requires a non-flag argument it is listed  i.e.  ```
var CommandArchiveCmd = &cobra.Command{
	Use:     "archive [commandID]",
```
There are a few in this file that do no conform.  @ethervoid   @mgdelacroix